src/mixins/browserCheck.js

```diff
@@ -65,19 +68,21 @@ const browserCheck = {
 		isFullySupported() {
 			return (this.isFirefox && this.majorVersion >= 52)
 			|| (this.isChrome && this.majorVersion >= 49)
+			|| (this.isOpera && this.majorVersion >= 36)
```

Comment:
So yeah tested and all fine… with Opera 73.
From my POV we can add opera, but then we shouldn't claim such an old version we are not testing with. I would just go with 72 for now.

Reply:
Ok, changed,